quaddtype/pyproject.toml

```diff
@@ -23,3 +22,61 @@ dependencies = [
 test = [
     "pytest",
 ]
+
+# [tool.cibuildwheel]
+# archs = ["auto64"]
+# skip = ["*-musllinux*", "pp*", "cp36-*", "cp37-*", "cp38-*", "cp39-*", "cp313-*"]
+# test-command = """
+# python -c "import platform; print('Python version:', platform.python_version())"
+# python -c "import sys; print('sys.platform:', sys.platform)"
+# python -c "import quaddtype; print('quaddtype imported successfully')"
+# pip install {package}[test]
+# pytest {project}/tests
+# """
+# test-extras = ["test"]
+# manylinux-x86_64-image = "manylinux_2_28"
+
+# [tool.cibuildwheel.linux]
+# before-all = '''
+# set -ex
+# yum install -y wget
+# wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+# bash miniconda.sh -b -p $HOME/miniconda
+# export PATH="$HOME/miniconda/bin:$PATH"
+# conda init bash
+# source ~/.bashrc
+# conda update -q conda
+# conda config --add channels conda-forge
+# conda config --set channel_priority strict
+# conda install -y -c conda-forge sleef
+# if [ ! -f "$HOME/miniconda/include/sleef.h" ]; then
+#     echo "sleef.h not found. Installation may have failed."
+#     exit 1
+# fi
+# ls -l $HOME/miniconda/include/sleef.h
+# ls -l $HOME/miniconda/lib/libsleef*
+# '''
+# environment = { LD_LIBRARY_PATH = "$HOME/miniconda/lib:$LD_LIBRARY_PATH", LIBRARY_PATH = "$HOME/miniconda/lib:$LIBRARY_PATH", CFLAGS = "-I$HOME/miniconda/include $CFLAGS", CXXFLAGS = "-I$HOME/miniconda/include $CXXFLAGS", LDFLAGS = "-L$HOME/miniconda/lib $LDFLAGS" }
+# repair-wheel-command = "auditwheel repair -w {dest_dir} --plat manylinux_2_28_x86_64 {wheel}"
+
+# [tool.cibuildwheel.macos]
+# before-all = '''
+# set -ex
+# ARCH=$(uname -m)
+# wget https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-${ARCH}.sh -O miniconda.sh
+# bash miniconda.sh -b -p $HOME/miniconda
+# export PATH="$HOME/miniconda/bin:$PATH"
+# source ~/.bash_profile
+# conda update -q conda
+# conda config --add channels conda-forge
+# conda config --set channel_priority strict
+# conda install -y -c conda-forge sleef
+# if [ ! -f "$HOME/miniconda/include/sleef.h" ]; then
+#     echo "sleef.h not found. Installation may have failed."
+#     exit 1
+# fi
+# ls -l $HOME/miniconda/include/sleef.h
+# ls -l $HOME/miniconda/lib/libsleef*
+# '''
+# environment = {DYLD_LIBRARY_PATH = "$HOME/miniconda/lib:$DYLD_LIBRARY_PATH", LIBRARY_PATH = "$HOME/miniconda/lib:$LIBRARY_PATH", CFLAGS = "-I$HOME/miniconda/include $CFLAGS", CXXFLAGS = "-I$HOME/miniconda/include $CXXFLAGS", LDFLAGS = "-L$HOME/miniconda/lib $LDFLAGS", SLEEF_INCLUDE_DIR = "$HOME/miniconda/include", SLEEF_LIBRARY = "$HOME/miniconda/lib", MACOSX_DEPLOYMENT_TARGET = "10.13"}
+# repair-wheel-command = "delocate-wheel -w {dest_dir} -v {wheel}"
```

Comment:
Why is all of this configuration commented out?

Reply:
I put the configuration from there to workflow itself, so not needed there anymore

Reply:
OK in that case you should delete it. IMO you should never commit commented out code to a public repo, it's always going to be a source of confusion and bitrotted code.

Reply:
Understood ✅